app/hotels/src/appRegistry/HotelsStandalonePackage.js

```diff
@@ -7,7 +7,7 @@ import RootComponent from './RootComponent';
 import HotelsStack from '../navigation/NavigationStack';
 import type { Coordinates } from '../CoordinatesType';
 
-type Props = {|
+type Props = {
```

Comment:
Why did you change this? Is it necessary? If yes, I am OK with that but otherwise it doesn't make sense.

Reply:
Yeah it is necessary for now or I would have to use a Flow comment. The `WithNativeNavigation` is injecting `onBackClicked` prop but in your non-native app, you are also using that wrapper that injects `onBackClicked` so it is a bit tricky to make back button works in this case for both not-native and native.

Flow does not seem to understand it:

```js
<WrappedComponent
  onBackClicked={
    GestureController.isNativeGestureModule ? this.onBackClicked : null
  }
  onNavigationStateChange={this.onNavigationStateChange}
  {...this.props}
/>
```

For the native app, it will use the `GestureController`. For the wrapper like `HotelsPackageWrapper`, it will be passing `onBackClicked` which overrides the `null` one with `{...this.props}`. Makes sense?